src/full/Agda/Syntax/Concrete/Name.hs

```diff
@@ -260,19 +260,29 @@ instance LensInScope QName where
 -- * Generating fresh names
 ------------------------------------------------------------------------
 
-nextRawName :: RawName -> RawName
-nextRawName s = addSuffix root (maybe initialSuffix nextSuffix suffix)
+-- | Method by which to generate fresh unshadowed names.
+data FreshNameMode
+  = UnicodeSubscript
+  -- ^ Append an integer Unicode subscript: x, x₁, x₂, …
+  | AsciiCounter
+  -- ^ Append an integer ASCII counter: x, x1, x2, …
+  | PrimeTickCount
```

Comment:
Is this every used?

Reply:
Ah, no. I was blindly making it correspond to the formats supported by `Agda.Utils.Suffix`, which has the additional `Prime Int` constructor. But that's only ever used when a name _already_ ends in an apostrophe, never for the initial permutation. (That distinction about prime ticked names existed before this PR, not as a change here).

I'll remove this unused constructor and add a comment noting that point.

Reply:
Pushed